pyocd/target/builtin/target_STM32L432xx.py

```diff
@@ -0,0 +1,102 @@
+# pyOCD debugger
+# Copyright (c) 2018-2019 Arm Limited
```

Comment:
Please change this to your copyright for 2020.

Reply:
do you mean Arm Limited should be my name on it?

Reply:
Your name (or employer, if appropriate), _not_ Arm Limited. You own the copyright since you created the file.

Reply:
fixed